.cargo/config.toml

```diff
@@ -0,0 +1,33 @@
+#
+# An auto defined `clippy` feature was introduced,
+# but it was found to clash with user defined features,
+# so was renamed to `cargo-clippy`.
+#
+# If you want standard clippy run:
+# RUSTFLAGS= cargo clippy
+[target.'cfg(feature = "cargo-clippy")']
+rustflags = [
+  "-Aclippy::all",
+  "-Dclippy::correctness",
+  "-Aclippy::if-same-then-else",
+  "-Aclippy::clone-double-ref",
+  "-Dclippy::complexity",
+  "-Aclippy::clone_on_copy",             # Too common
```

Comment:
So many -A clippy, this is just added just for runnig  cargo clippy passed ?

Reply:
These rules are copied from polkadot and substrate, I think we can keep up with the official style, if we turn to default strict mode then too many changes need to do.